get-name/src/main/java/com/example/getFieldName/repository/NameRepository.java

```diff
@@ -0,0 +1,5 @@
+package com.example.getFieldName.repository;
+import com.example.getFieldName.entity.NameEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface NameRepository extends JpaRepository<NameEntity, Long>{}
```

Comment:
Do you have onle name in your repository?? should it be Student repositoty where you get some name?